src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -189,6 +190,41 @@ public static void DefaultSettings_Echo_RoundTrips_String_Streamed_Async_WithSin
         Assert.True(success, "Test Scenario: DefaultSettings_Echo_RoundTrips_String_Streamed_Async_WithSingleThreadedSyncContext timed-out.");
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void ServiceContract_EchoMessageParameter()
+    {
```

Comment:
We need a method name that describes what the test does.  Something like "ServiceContract_Call_Operation_With_MessageParameterAttribute"